packages/frontend/app/components/app/sidebar/index.ts

```diff
@@ -38,4 +41,25 @@ export default class Sidebar extends Component {
     scrollIntoViewOfParent(scrollable, lastRow);
     this.sidebar.clearUnreadAbove();
   }
+
+  @action
+  switchToContacts() {
+    this.contactsSelected = true;
```

Comment:
Would an enum-like object be better here? Have one true state:tm: rather than potentially every tab active?

Reply:
Maybe a 'switchToTab' action?

Reply:
I would also think this would be the way to go, as this is essentially a state switcher between three states.

Reply:
we tried using an enum but couldn't get it working with handlebars

Reply:
how do you mean?